lib/msal-browser/src/app/ClientApplication.ts

```diff
@@ -324,8 +384,23 @@ export abstract class ClientApplication {
             ThrottlingUtils.removeThrottle(this.browserStorage, this.config.auth.clientId, authCodeRequest.authority, authCodeRequest.scopes);
 
             // Handle response from hash string.
-            return await interactionHandler.handleCodeResponse(hash);
+            const result = await interactionHandler.handleCodeResponse(hash);
+
+            // If logged in, emit acquire token events
+            if (isLoggedIn) {
```

Comment:
We should check against the difference between the original account state and current state here to determine if an additional user was logged in or not